src/extension/monitor/core/src/main/java/org/geoserver/monitor/MonitorConfig.java

```diff
@@ -130,6 +132,15 @@ public BboxMode getBboxMode() {
         return BboxMode.valueOf(mode.toUpperCase());
     }
 
+    public List<String> getIgnorePostProcessors() {
```

Comment:
Since we care about containment check, best return a Set instead.

Reply:
understood, will change to Set<String>